src/demo/examples/form/index.js

```diff
@@ -21,6 +21,17 @@ export default () => {
     .addPicker('Picker', ['red', 'green', 'blue'], { required: true })
     .addTagInput('Tag Input')
     .addTagInput('Tag Input', { required: true })
+    .addTagInput('Tag Input', {
+      required: true,
+      tagInputOptions: {
+        validator: (name) => {
+          if (!isNaN(Number(name))) {
```

Comment:
This is causing the lint failure in travis - Use `!Number.isNaN(Number(name))` as `isNaN(...)` is generally bad practice (see [AirBnB Docs](https://github.com/airbnb/javascript#standard-library--isnan))